src/asm-manual.adoc

```diff
@@ -1027,15 +1073,15 @@ s{b\|h\|w\|d} rd, symbol[11:0](rt)
 |Store global
 |
 
-|fl{w\|d} rd, symbol, rt
+|fl{h\|w\|d} rd, symbol, rt
 | auipc rt, symbol[31:12] +
-fl{w\|d} rd, symbol[11:0](rt)
+fl{h\|w\|d} rd, symbol[11:0](rt)
 |Floating-point load global
 |
 
-|fs{w\|d} rd, symbol, rt
+|fs{h\|w\|d} rd, symbol, rt
 |auipc rt, symbol[31:12] +
-fs{w\|d} rd, symbol[11:0](rt)
+fs{h\|w\|d} rd, symbol[11:0](rt)
```

Comment:
the `flq` / `fsq` pseudo is also supported by LLVM's implementation of the Q extension.

Reply:
@apazos